workflow-templates/zephyr/README.md

```diff
@@ -0,0 +1,180 @@
+# Zephyr
+
+## Description
+
+Workflows in this group extend the workflows in [Basic](../basic/README.md)
+group and introduce several new functionalities which aid in the development of
+the projects using Zephyr.
+
+Specifically, they provide:
+
+- Everything that [Basic](../basic/README.md) workflows already do: changelog
+  preparation, tagging, publishing GitHub releases, etc.,
+- Running _builds_ on Pull Requests and during releases processes,
+- Caching of West modules and toolchains downloaded by East to speed up the
+  _build_ process,
+- A way to change what _build_ means for each project,
+- A way to specify which artefacts are attached to the published GitHub releases
+  for each project,
+- A tag and Changelog cleanup steps when _build_ goes wrong and
+- A way to add custom text to the release notes.
+
+## Dependencies
+
+Workflows in this group expect specific files to be present in the repository to
+function properly. If you created your repository from `irnas-zephyr-template`
+then you are all set with basic defaults.
+
+Needed files (relative to project's root dir):
+
+- `scripts/requirements.txt`, can be empty,
+- `makefile` - Specific content is expected, see section
+  [How to configure _build_](#how-to-configure-build).
+
+### How to use
+
+This group contains four workflows:
+
+- `create-release.yaml`
+- `publish-release.yaml`
+- `build.yaml`
+- `label_pr.yaml`
+
+They can be used in two different scenarios:
+
+- During a release process
+- In Pull Requests
+
+#### Release process
+
+To trigger a release process just follow the instructions in the Basic's
+[How to use](../basic/README.md#how-to-use) section.
+
+Everything that is described in that section still applies, with some
+modifications:
+
+- After creating a release tag and a new changelog section a `build.yaml`
+  workflow is called.
+- `build.yaml` runs a _build_ process.
+- After that `publish-release.yaml` takes any resulting _build_ artifacts and
+  creates a GitHub release with them.
+
+If anything goes wrong during `build.yaml` and `publish-release.yaml` workflows
+then the created release tag and Changelog update commit are deleted from the
+`main` branch.
+
+#### Pull requests
+
+`build.yaml` (and therefore _build_ process) is automatically triggered whenever
+a PR is opened, reopened or a new commit is pushed to it.
```

Comment:
It triggers when a commit is pushed to the existing PR, right? Try wording this a bit more clearly.